### DISCUSSION TITLE
Notation does not replicate the artifact to another harbor replica to verify the image signing

### DISCUSSION THREAD

**Discussion by TnGadaria** [goharbor/harbor#20733] [Q&A] [answered]
Harbor version: 2.11.0

Hi,
I am trying out notation to sign the image and I do have another harbor registry as configured as a replica, however, the sign artifact is not replicated, and hence I am not able to verify the image sign. I went through a couple of issues [16679](https://github.com/goharbor/harbor/issues/16679 ), but this is to confirm if the feature is introduced in the harbor 2.11.0 harbor release. ? 

I was trying out cosign as well. We have the LDAP-based auth configured for users, and we noticed the cosign does take the creds from the docker creds store during login, however, it does not allow us to sign/verify the images. If we sign the image via cosign as an admin user, it works fine. 
Also, the admin user does not sign the image pushed in the private registry, whereas the admin is already a member of the registry. 

Can someone help me with this?

**Comment by wy65701436** [comment, marked as answer]
1, can you confirm the version of your notation? Please double check with the FAQ: https://github.com/goharbor/harbor/wiki/Harbor-FAQs#notation-signature-accessory.
2, are you saying that you cannot sign an image via cosign with an non-admin user?

**Reply by TnGadaria** [reply]
Hi @wy65701436 

1. I am using notation version `v1.1.0`. After your post, I tried with the notation `v1.2.0-alpha.1` and it works like a charm.
2. Yes, I used Cosign earlier and faced the following problem. (LDAP Auth is integrated with Registry). If I log in via admin creds using cosign that works, however, if I use LDAP for authentication, it does not allow me to sign the image, and it throws me out with `Unauthorised to access the project`. Though I was a project member.
